packages/website/components/GlobalStyles.tsx

```diff
@@ -28,8 +28,8 @@ export const GlobalStyles = () => {
         body {
           color: ${tokens.gray800};
           font-family: ${tokens.fontStackPrimary};
-          font-size: ${tokens.fontSizeL};
-          line-height: ${tokens.lineHeightL};
+          font-size: ${tokens.fontSizeM};
+          line-height: ${tokens.lineHeightM};
```

Comment:
Do we really want to have code in the code blocks smaller? For me, it's a bit inconvenient right now to read it.

Reply:
Yeah, that was the requirement from design.